src/Pages/AccountPage/SignUp.scss

```
@@ -0,0 +1,222 @@
+.signUpHome {
+  display: flex;
+  justify-content: center;
+  .icon {
```

```suggestion
  justify-content: center;
  
  .icon {
```
새로운 셀렉터 앞에서는 가독성을 위해 한줄 띄워주세요